Specification.md

```diff
@@ -1,17 +1,145 @@
-## Gas limit
+# Specification
 
-By default `web3.js` uses `web3.eth.Contract.method.myMethod().estimateGas()`, but this value can differ from the 
-actual gas that will be used, if the contract you are calling depends on the blockhash, blocknumber or any other source of 
-randomness making the gas cost nondeterministic.
+The CLI should package several "core" extensions.
+The API these extensions expose is aimed at `node` environments.
+The aim is to provide convenience to power-users & devs.
 
-E.g.: the [`MiniMeToken`](https://github.com/aragon/aragon-apps/blob/master/shared/minime/contracts/MiniMeToken.sol) contract which snapshots balances at certain block numbers.
+## IPFS `@aragon/ipfs-utils`
```

Comment:
For extensions related to the cli, perhaps it would make sense to prefix them with `cli` (so this would become `@aragon/cli-ipfs-utils`)?

Reply:
Hmm, makes sense.
What about `dao`, `apm`, etc? I think we can publish those without the `utils` suffix, e.g.: `@aragon/cli-dao`.

Reply:
I like `@aragon/cli-<package>` format